lib/ui/screens/protected/resturant-store/restaurant/index.tsx

```
@@ -66,14 +78,22 @@ export default function RestaurantDetailsScreen() {
-            ...food,
-            title: food.title.toLowerCase(), // Modify food titles if needed
-          })),
-        })) || []
+          foods: c.foods.filter(food => {
+            // If filter is empty, include all foods
+            if (filter.trim() === "") return true;
+            
+            // Include food if title or description matches filter
+            return (
+              food.title.toLowerCase().includes(filter.toLowerCase()) ||
+              (food.description && food.description.toLowerCase().includes(filter.toLowerCase()))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)